atomic_defi_design/qml/Components/CexInfoTrigger.qml

```diff
@@ -5,6 +5,18 @@ import QtQuick.Controls 2.15
 import "../Constants"
 
 DefaultMouseArea {
+    id: mouse_area
     anchors.fill: parent
     onClicked: cex_rates_modal.open()
+    hoverEnabled: true
+
+    DefaultTooltip {
+        visible: mouse_area.containsMouse
+
+        delay: 500
+
+        contentItem: DefaultText {
+            text_value: "Price oracle powered by Band Protocol"
```

Comment:
Usage of https://doc.qt.io/qt-5/qtquick-internationalization.html#1-use-qstr-for-all-literal-user-interface-strings is required here i think.

Reply:
Yes, please use `qsTr(...)` so the text can be translated.